src/app/core/config/config-fix.ts

```diff
@@ -256,7 +256,10 @@ export const defaultJsonConfig = {
     "lazyLoaded":  true
   },
   "view:help": {
-    "component": "Help"
+    "component": "MarkdownPage",
+    "config": {
+      "markdownFile": $localize`:Filename of markdown help page:assets/help/help.en.md`,
```

Comment:
Maybe there should be a small note that this is not supposed to be 'translated' other than by someone that actually knows where this file is?

Reply:
done :+1: